plugins/modules/system/parted.py

```diff
@@ -610,7 +619,8 @@ def main():
 
         # Create partition if required
         if part_type and not part_exists(current_parts, 'num', number):
-            script += "mkpart %s %s %s " % (
+            # GNU parted <= 3.2.153 bug forced (optional) filesystem parameter for negative partition start
+            script += "mkpart %s ext2 %s %s " % (
```

Comment:
Now 'ext2' fstype is coded inline, but after #135 it will be default parameter value